RunningOrder/Sprint/Managers/SprintManager.swift

```diff
@@ -60,6 +60,25 @@ final class SprintManager: ObservableObject {
         }
     }
 
+    func deleteSprint(_ sprint: Sprint) {
```

Comment:
there is lack of consistency between `deleteSprint(_:)` & `delete(sprint:)`. Could you pease fix that? :)